### DISCUSSION TITLE
使用wired_table_rec模式中文字芫解析成荒，请问应该如何解决

### DISCUSSION THREAD

**Discussion by weiwan520** [RapidAI/TableStructureRec#12] [Q&A]
使用wired_table_rec模式解析中文有线表格时，中文字芫解析成荒，请问应该如何解决

**Comment by SWHL** [comment]
这个是因为依赖的OCR模块中文字识别错误导致，这种问题一般要微调文本识别模型了。暂时没有其他简单的方法。
